src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -157,13 +151,6 @@ public function start()
             throw new \RuntimeException('Failed to start the session.');
         }
 
-        if (null !== $this->emulateSameSite) {
-            $originalCookie = SessionUtils::popSessionCookie(session_name(), session_id());
```

Comment:
is this `SessionUtils::popSessionCookie` utility method still necessary then or should it also be removed ?

Reply:
It’s still used elsewhere.